ios/browser/api/sync/brave_sync_api.h

```diff
@@ -58,6 +88,11 @@ OBJC_EXPORT
 
 - (void)resetSync;
 
+- (void)didJoinSyncChain:(void (^)(bool))completion;
```

Comment:
Should be a property, if it can't be a property then we need `set` prefix

Reply:
Fix at https://github.com/brave/brave-core/pull/17369/commits/78e12f9fa358d9d860f5d857a20d03b4e707a54c